internal/sqlquery/scraper.go

```diff
@@ -107,3 +109,142 @@ func (s *Scraper) Shutdown(_ context.Context) error {
 	}
 	return nil
 }
+
+func BuildDataSourceString(driver string, dataSource DataSourceConfig) (string, error) {
+	switch driver {
+	case "postgres":
```

Comment:
I think we may want to move all references to driver names to constants, and reference those instead of plaintext strings all over.